examples/create-consensus-topic.js

```diff
@@ -0,0 +1,29 @@
+const { Client, ConsensusTopicCreateTransaction } = require("@hashgraph/sdk");
+
+async function main() {
+  const operatorAccount = process.env.OPERATOR_ID;
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = new Client({
+    network: { "0.testnet.hedera.com:50211": "0.0.3" },
+    operator: {
+      account: operatorAccount,
+      privateKey: operatorPrivateKey
+    }
+  });
+
+  const tx = await new ConsensusTopicCreateTransaction().execute(client);
+  console.log("tx:", tx);
+
+  const record = await tx.getRecord(client);
+  const newTopicId = record.receipt._topicId;
```

Comment:
`_topicId` should be `.getTopicId()`

---

For context this changed to match Java _and_ to provide a tiny bit of debugging help so if you call `.getFileId()` where the was no file ID it will throw an exception with more information besides just emitting "undefined".